k6script.js

```diff
@@ -0,0 +1,37 @@
+import { check, sleep } from 'k6';
+import http from 'k6/http';
+import idStrings from './k6RandomIds.js';
```

Comment:
did you end up making use of your random IDs?  wasn't able to find in here.

Reply:
These are only used for the 'get items by area of ids' situation.  I've been using the same k6 file for all stress tests, making small changes to the route as necessary.